src/commands/analytics/log-flow.ts

```diff
@@ -0,0 +1,138 @@
+import { AppCommand, CommandResult, ErrorCodes, failure, hasArg, help, longName, shortName, success} from "../../util/commandline";
+import { MobileCenterClient, models, clientRequest, ClientResponse } from "../../util/apis";
+import { StreamingArrayOutput } from "../../util/interaction";
+import { inspect } from "util";
+import * as _ from "lodash";
+import { DefaultApp } from "../../util/profile";
+import * as ContinuousPollingHelper from "../../util/continuous-polling/continuous-polling-helper";
+
+const debug = require("debug")("mobile-center-cli:commands:analytics:log-flow");
+
+@help("Command to see the incoming logs in real time")
+export default class ShowLogFlowCommand extends AppCommand {
+  private static readonly delayBetweenRequests = 3000;
+
+  @help("Introduce the number of logs (max 100) that are being displayed, default number is 100")
+  @shortName("l")
+  @longName("num-logs")
+  @hasArg
+  public logsCount: string;
+
+  @help("Filter the logs by install ID")
+  @shortName("i")
+  @longName("install-id")
+  @hasArg
+  public installationId: string;
+
+  @help("Continue to return logs, press Ctrl+C to exit")
+  @shortName("c")
+  @longName("continue")
+  public showContinuously: boolean;
+
+  public async run(client: MobileCenterClient): Promise<CommandResult> {
+    const app: DefaultApp = this.app;
+
+    this.validateParameters();
+
+    const logsCount = Number(this.logsCount) || 100;
+
+    const streamingOutput = new StreamingArrayOutput();
+    streamingOutput.start();
+
+    let options: {start: Date};
+    await ContinuousPollingHelper.pollContinuously(async (requestsDone: number) => {
+      try {
+        debug ("Loading logs");
+        // start time is not specified for the first request
+        return await clientRequest(
+          (cb) => requestsDone ? 
+            client.analytics.genericLogFlow(app.ownerName, app.appName, options, cb) : 
```

Comment:
I'm pretty sure if you leave options `undefined` or `null` it'll act the same as if you didn't pass it at all, so I don't think you need the `requestsDone ?` condition here.

Yeah, I just double checked the client code - just initialize options to `null` the first time through and pass it in, everything will work.